src/outagesim/backup_reliability.jl

```diff
@@ -434,20 +697,21 @@ Return a dictionary of inputs required for backup reliability calculations.
     -gen_operational_availability::Real = 0.9998        Fraction of year generators not down for maintenance
     -gen_failure_to_start::Real = 0.0066                Chance of generator starting given outage
     -gen_failure_to_run::Real = 0.00157                 Chance of generator failing in each hour of outage
-    -num_gens::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
-    -gen_capacity::Real = 0.0                           Backup generator capacity. Will be determined by REopt optimization if set less than 0.1
+    -num_gen::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
+    -gen_capacity_kw::Real = 0.0                           Backup generator capacity. Will be determined by REopt optimization if set less than 0.1
```

Comment:
In addition to gen, ERP user also needs to be able to specify PV and CHP size, not just taken from REopt results. Also, when you make the changes to this function that I requested in #r962108891 PV and CHP prod series inputs will need to be normalized to size!